src/Abstractions/Internal/IOrchestrationSubmitter.cs

```diff
@@ -40,5 +41,6 @@ public interface IOrchestrationSubmitter
         TaskName orchestratorName,
         object? input = null,
         StartOrchestrationOptions? options = null,
+        HashSet<string>? orchestrationIdReusePolicy = null,
```

Comment:
Please add this to `StartOrchestrationOptions` instead of its own parameter. All non-essential parameters for affecting orchestration start should go there.

Reply:
Also, should this be an enum with the available behaviors as values in it?